Section 5 -- Validation.md

```diff
@@ -425,7 +425,7 @@ fragment goodNonNullArg on Arguments {
 
 ```
 
-On a field with a a nullable arg, that argument can be omitted.
+A field with a nullable argument can be omitted.
```

Comment:
This seems to change the meaning, suggesting that the field can be omitted (rather than that the argument can be omitted)?